messaging/src/main/java/org/axonframework/queryhandling/SimpleQueryBus.java

```diff
@@ -231,7 +232,7 @@ private <Q, R> CompletableFuture<QueryResponseMessage<R>> doQuery(
 
     @Override
     public <Q, R> Publisher<QueryResponseMessage<R>> streamingQuery(StreamingQueryMessage<Q, R> query) {
-        Span span = spanFactory.createChildHandlerSpan(() -> "SimpleQueryBus.streamingQuery", query).start();
+        Span span = spanFactory.createStreamingQuerySpan(query, false).start();
```

Comment:
Curious: Why did this change from a child handle span format to an internal span?

Reply:
Good question. I know I did it for *a* reason, but I cannot remember it. I will revert the change unless we come up with a good argument,